File: MyHeart.Web/ClientApp/models/professionInfo/MedicamentGroupDto.ts
import MedicamentGroupDiseaseContraindicationDto from "./bonds/MedicamentGroupDiseaseContraindicationDto";
import MedicamentGroupDiseaseIndicationDto from "./bonds/MedicamentGroupDiseaseIndicationDto";
import MedicamentGroupPharmacyDto from "./bonds/MedicamentGroupPharmacyDto";
import MedicamentGroupSymptomsSideEffectsDto from "./bonds/MedicamentGroupSymptomsSideEffectsDto";
import MedicamentGroupAtcsDto from "./bonds/MedicamentGroupAtcsDto"
import PharmacyDto from "./PharmacyDto";

export default class MedicamentGroupDto {
    id: number;
    name: string | null;
    description: string | null;
    contraindication: MedicamentGroupDiseaseContraindicationDto[];
    indication: MedicamentGroupDiseaseIndicationDto[];
    activeSubstance: MedicamentGroupPharmacyDto[];
    sideEffects: MedicamentGroupSymptomsSideEffectsDto[];
    atcs: MedicamentGroupAtcsDto[];
    pharmacies: PharmacyDto[];
}

File: MyHeart.DTO/ProfessionInformation/MedicamentGroupDTO.cs
using MyHeart.DTO.ProfessionInformation.MedicamentGroupBonds;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace MyHeart.DTO.ProfessionalInformation
{
    public class MedicamentGroupDTO : BaseEntityDTO
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public List<MedicamentGroup_Disease_ContraindicationDTO> Contraindication { get; set; }
        public List<MedicamentGroup_Disease_IndicationDTO> Indication { get; set; }
        public List<MedicamentGroup_Pharmacy_ActiveSubstanceDTO> ActiveSubstance { get; set; }
        public List<MedicamentGroup_Symptoms_SideEffectsDTO> SideEffects { get; set; }
        public List<MedicamentGroup_AtcDTO> Atcs { get; set; }
        public List<PharmacyDTO> Pharmacies { get; set; }
    }
}
